infrastructure/Dockerfile

```diff
@@ -8,8 +8,8 @@ ARG CORFU_TOOLS_JAR
 WORKDIR /app
 
 RUN apt update && apt -y install iptables bash jq python3 sudo iproute2 python3-pip curl
-RUN python3 -m pip install pyyaml==6.0.1
-RUN python3 -m pip install netifaces==0.11.0
+RUN apt-get install -y python3-yaml
```

Comment:
As suggested by Coday, is it better to add the version info?  
apt-get install <package>=<version>

Reply:
Done, Thank you!

Reply:
On Noble Numbat images (Ubuntu 24), apt distribution has these versions so we are good to push without version pinning.
```
python3-netifaces is already the newest version (0.11.0-2build3).
python3-yaml is already the newest version (6.0.1-2build2).
```